code/game/objects/items/weapons/defib.dm

```diff
@@ -243,12 +243,15 @@
 	paddle_type = /obj/item/twohanded/shockpaddles/advanced
 	combat = TRUE
 	safety = TRUE
-	hardened = TRUE  // emp-proof (on the component), but not emag-proof.
+	hardened = TRUE // emp-proof (on the component), but not emag-proof.
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF //Objective item, better not have it destroyed.
 	heart_attack_probability = 10
 
 	var/next_emp_message //to prevent spam from the emagging message on the advanced defibrillator
 
+/obj/item/defibrillator/compact/advanced/screwdriver_act(mob/living/user, obj/item/I)
```

Comment:
The non-`loaded` variant doesn't actually have a power cell inside it, so putting this here is a bit overzealous
On the other hand, I suppose it doesn't really matter if this override is on the subtype or the main type considering that you can't actually insert power cells into advanced defibs due to the random `attackby` override